doc/release-notes/0.20/0.20.md

```diff
@@ -98,6 +98,49 @@ The v0.20.0 release contains the following known issues and limitations:
 </thead>
 <tbody>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/8034">#8034</a></td>
+<td valign="top">Balanced garbage collection policy not supported</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">The Balanced GC policy cannot be used. </td>
+<td valign="top">Use an alternative GC policy, such as <tt>gencon</tt>.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/8762">#8762</a></td>
+<td valign="top">Concurrent Scavenge mode not supported</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">The concurrent scavenge mode (<tt>-Xgc:concurrentScavenge</tt>) of the <tt>gencon</tt>
+garbage collection policy cannot be used. </td>
+<td valign="top">The <tt>gencon</tt> policy can still be used without this mode operational.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5917">#5917</a></td>
+<td valign="top">JIT dynamic loop transfer not supported</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">Performance is not optimized in this release. </td>
+<td valign="top">None. Performance will improve in a future release.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/6607">#6607</a></td>
+<td valign="top">JIT recompilation not supported</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">Performance is not optimized in this release. </td>
+<td valign="top">None. Performance will improve in a future release.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/8966">#8966</a>/<a href="https://github.com/eclipse/openj9/issues/9003">#9003</a></td>
+<td valign="top">AOT compilation is not working by default</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">When using shared classes, startup performance is not optimized in this release. </td>
+<td valign="top">To turn on AOT explicitly, specify the <tt>-Xquickstart</tt> or <tt>-Xaot</tt> option.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/9251">#9251</a></td>
+<td valign="top">Problems reading system dumps</td>
+<td valign="top">Linux on 64-bit ARM (AArch64)</td>
+<td valign="top">Although core files are successfully generated on AArch64 systems, the dump viewer (<tt>jdmpview</tt>) in an Aarch64 JVM is unable to read them.</td>
```

Comment:
I would like words like "in some cases" to be added here.  It does not always fail, depending on unknown condition.

Reply:
Done